app/Svg/Builder.hs

```diff
@@ -192,7 +209,7 @@ shapeTolerance s =
 -- Invert a 3x3 matrix. Assumes that the matrix is invertible
 invertMatrix3x3 :: Matrix -> Matrix
 invertMatrix3x3 m =
-    map (map (* (1 / determinantMatrix3x3 m))) (transposeMatrix3x3 (cofactorMatrix3x3 m))
+    transposeMatrix3x3 $ map (map (* (1 / determinantMatrix3x3 m))) (transposeMatrix3x3 (cofactorMatrix3x3 m))
```

Comment:
You've correctly identified the bug (fantastic!) but in terms of fixing it, there is an underlying issue.

First, since the main operation on this line is multiplying a matrix by a scalar, the two transposes cancel each other out, and so it is more elegant to remove the existing `transmoseMatrix3x3` call rather than add another one.

Second, the underlying issue is that `cofactorMatrix3x3` isn't actually returning the cofactor matrix, but rather the [adjoint matrix](https://en.wikipedia.org/wiki/Adjugate_matrix#3_%C3%97_3_generic_matrix) (which is the transpose of the cofactor matrix, which explains the "extra" transpose in the existing code). This is the underlying error in the formula. Please rename the function to `adjointMatrix3x3`.